### PR TITLE
Closes #120 . For real this time.  Remove udunitspy words

### LINKED ISSUE #120: Remove references to udunits from docs
README, ubuntu install guide, and various comments in code still refer to udunits and trouble surrounding it.

As of #114 we no longer use udunits, so these need to be removed.

### PR DESCRIPTION
**Pull request by DanielJMaher**
Leaves references to udunits, as it should. 

@lukecampbell @daf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -394,7 +394,7 @@ conda install compliance-checker
 
 ### MS-Windows
 
-IOOS Compliance Checker cannot be directly installed on MS-Windows systems because the required `udunitspy` and `swig` packages are either broken or unavailable for that OS (2014-08-18). However, the Compliance Checker can be successfully installed on Linux that runs as a guest OS on a virtual platform, e.g. Oracle VirtualBox with Linux Mint. If Anaconda is the Python of choice for the virtual Linux machine, the 64bit Linux has to be used because no `udunitspy` is currently available for 32bit Linux.
+IOOS Compliance Checker cannot be directly installed on MS-Windows systems because the required and `swig` package are either broken or unavailable for that OS (2014-08-18). However, the Compliance Checker can be successfully installed on Linux that runs as a guest OS on a virtual platform, e.g. Oracle VirtualBox with Linux Mint.
```
Comment (ocefpaf) [inline README.md]: I think that is no longer true.

Comment (daf) [inline README.md]: this isn't english, nor is swig required because it was what udunitspy needed

**Comment by DanielJMaher** [comment]
@daf removed swig talk

**Comment by daf** [comment]
also from doc/ubuntu-install-guide.md plz

**Pull request merged**